src/com/google/enterprise/adaptor/prebuilt/SkipDocumentFilter.java

```diff
@@ -203,19 +202,9 @@ public void transform(Metadata metadata, Map<String, String> params) {
             + corpora);
         params.put(MetadataTransform.KEY_TRANSMISSION_DECISION,
             TransmissionDecision.DO_NOT_INDEX.toString());
-      } else {
-        log.info("Not skipping document " + docId + ", because we did not find "
```

Comment:
Why did you choose not to preserve the log messages? What about keeping them at FINE or lower?